### ISSUE TITLE
Rewrite and update Chapter 1 of the July 2019 Monterrey tutorial to run with MET9/METplus3 on hera and dakota

### ISSUE THREAD

**Issue by fentoad72** [dtcenter/METplus#408]
component: documentation

**Comment by georgemccabe** [comment]
I reviewed the content of Session 1 and it looks good.

**Comment by georgemccabe** [comment]
@j-opatz , could you run through the instructions on dakota and make sure that everything in Chapter 1 is accurate and works correctly? You can start from here:

https://dtcenter.org/metplus-practical-session-guide-2020-beta/session-1-metplus-setupgrid-grid

**Comment by j-opatz** [comment]
I completed a review of chapter one, and have the following notes:

- METplus Setup
   - Under the Directories and Configuration Files - Overview, in the Example use case call, ${METPLUS_TUTORIAL_DIR} is incorrect; it should use ${METPLUS_BUILD_BASE}, as the TUTORIAL_DIR is just the local holder for user files.
      - Also note that the path needs a parm after the ${} command; use_cases is not a top level directory
   - In the METplus How To Run section, it looks like each file and forecast time was processed twice by Example? And five date-times were processed, not the four as stated.

- MET tool: PCP Combine
   - On the splash page for PCP-Combine, PCP-Combine also supports Derive, as noted in the Add and Subtract section as well as its own section; however, the intro page for PCP-Combine does not list Derive.

- MET tool: Grid-Stat
   - In the Configure section, The link to the MET user’s guide goes to the 8.1 version. 
      - There may be additional times when the link is used that it goes to the older version before this, but this is where I caught it first.
   - In the Configure section, when running the gv ${MET_BUILD_BASE}/data/poly/ncep_vx_regions.pdf & command, the error message says the pdf does not exist in /usr/local/met-9.0/data/poly……looks like it actually resides in /usr/local/met-9.0/share/met/poly.
   -In the Configure section, in the blue section for setting output_flags, there is a missing dmap flag that should be added

- METplus Use Case
   - The MET users guide links to the 8.1 version, and the METplus user’s guide link doesn’t work.
   - The code to review the output files doesn’t appear to be in the correct formatting (web display, the command itself is fine)

- Additional Exercises
   - The answers say to modify “g2g.add_rh.conf”, but the name for the Grid-Stat exercise was named GridStat_add_rh.conf
   - The RH example has an ls -l command for the log file which does not exist
   - During the second example, the code is no longer highlighted with the grey boxes

- General Comments and Suggestions
   - As was discovered through troubleshooting, the advice to add the /bin to your PATH if a call to point-stat doesn't work should be explained with a quick explanation and command lines. The updated .sh file helps, but it seems that the additional steps of adding bin to your PATH and updating the python call (listed below) aren't possible needed fixes, they're mandatory to get it to work.
   -When discussing the METplus directory structure, docs/doxygen is listed, but it's a subdirectory. Maybe it shouldn't/doesn't need to be listed?
      - On that note, should all top level items be listed? (Aka requirements.txt, environment.yml, and check_python.py)
   - At the point where we are passing config files to master_metplus.py, users on Dakota will hit an impasse, as the default Python is 2.7.13, and METplus requires 3.6.3 or higher to run. The tutorial needs to be updated with how to get python 3.6.3 or higher used. Adding the metplus_env directory to my path worked for me very well, but I believe this will be limited to met-dev members.
   - When the tutorial guides the user through the log file for Example.conf, is states that the tool ran 4 times. It would probably be beneficial to show the user how they can see that it ran 4 times, rather than just accepting it as fact.
   - ncview does not seem to be a native command under default users. Might need to add a quick code suggestion to the guide for users to add it to their .bashrc files so they can access it.
   - gv took an extremely long time to load (may have been a VPN exasperated issue?). This might be something to mention in the tutorial (aka if gv doesn’t open right away, be patient; its load can be long).

**Comment by georgemccabe** [comment]
@j-opatz  I made the changes you listed for Chapter 1. Regarding the comment about adding /bin (actually ${MET_BUILD_BASE}/bin) to the path, I added that directory to the path in the tutorial setup script, so that shouldn't be a problem. We may have already discussed that previously. 

I added the metplus conda environment to the path. I think this SHOULD work for people not members of met-dev, but we should definitely test that out. I can ask Lisa to try it. 

ncdump is actually available in the conda bin directory, so if they can access that version of python, they will get ncdump too.
